pages.ja/common/7z.md

```diff
@@ -1,36 +1,36 @@
 # 7z
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバです。
+> 詳しくはこちら: <https://www.7-zip.org>
```

Comment:
We need a period here
```suggestion
> 詳しくはこちら: <https://www.7-zip.org>.
```

Reply:
Thank you for pointing it! Is this period necessary for TLDR to parse markdown correctly?

Reply:
It's not required for the markdown, but it's required to keep a format without lints. We don't have a lint for ja so we should do this manually.

Reply:
I understand it, I've added period to Japanese translation of `7z*`